src/nyc_trees/apps/users/urls/group.py

```diff
@@ -34,11 +34,10 @@
         name='edit_user_mapping_priveleges'),
 
     # Keep in sync with src/nyc_trees/js/src/reserveBlockfacePage.js
-    url(r'^(?P<group_slug>[\w-]+)/request-individual-mapper-status/$',
+    url(r'^(?P<group_slug>[\w-]+)/request-trusted-mapper-status/$',
```

Comment:
Does ``trusted mapper`` == ``individual mapper``? If so, this more correct name may actually cause confusion.